packages/core/examples/colorbar/index.ts

```diff
@@ -0,0 +1,728 @@
+import vtkColormaps from '@kitware/vtk.js/Rendering/Core/ColorTransferFunction/ColorMaps';
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  volumeLoader,
+  getRenderingEngine,
+  ui,
+} from '@cornerstonejs/core';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addDropdownToToolbar,
+  setCtTransferFunctionForVolumeActor,
+  setPetColorMapTransferFunctionForVolumeActor,
+} from '../../../../utils/demo/helpers';
+
+const { ViewportColorBar } = ui.widgets.colorbar;
+const { ColorBarScalePosition } = ui.widgets.colorbar.Enums;
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  PanTool,
+  WindowLevelTool,
+  StackScrollMouseWheelTool,
+  ZoomTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { MouseBindings } = csToolsEnums;
+
+const pause = (interval) =>
+  new Promise((resolve) => setTimeout(resolve, interval));
```

Comment:
requires a comment why we need it. The example should be bare minimum without any extra functions